changelog.md

```diff
@@ -6,6 +6,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ---
 
+## [0.13.10] - 2019-12-17
```

Comment:
btw i think we probably want to hold off on publishing/changelog changes until after a pr is approved yeah? otherwise we can get into a merge conflict here

Reply:
let me know how the docs go for ya, I'm sure I missed things 😭